examples/CMakeLists.txt

```diff
@@ -44,3 +35,9 @@ ADD_EXAMPLE(tagger tag-bilstm)
 ADD_EXAMPLE(variational-autoencoder rnnlm-aevb)
 ADD_EXAMPLE(word-embedding tok-embed)
 ADD_EXAMPLE(xor xor)
+
+if(ENABLE_BOOST)
+  ADD_EXAMPLE(multiprocessing rnnlm-mp)
```

Comment:
I'm not sure if we need a `NOT MSVC` statement here, maybe the CI can help here ;)

Reply:
You probably do, and I'm guessing CI just didn't catch it.

Reply:
Exactly, CI for Windows won't build the c++ examples (https://github.com/clab/dynet/pull/914), so I added the `if not` case now.